src/main/java/org/lantern/SystemTrayImpl.java

```diff
@@ -269,14 +269,13 @@ public void onUpdate(final UpdateEvent update) {
     }
 
     @Subscribe
-    public void onConnectivityStateChanged(final ProxyConnectionEvent csce) {
-        final ConnectivityStatus cs = csce.getConnectivityStatus();
-        log.debug("Received connectivity state changed: {}", cs);
+    public void onConnectivityStateChanged(final ProxyConnectedEvent csce) {
+        log.debug("Proxy connected");
         if (!this.model.getSettings().isUiEnabled()) {
             log.info("Ignoring event with UI disabled");
             return;
         }
-        onConnectivityStatus(cs);
+        onConnectivityStatus(ConnectivityStatus.CONNECTED);
```

Comment:
Again, once the proxy is connected, that's it.  We don't learn about any other statuses for proxies anymore.